kernel/common/enumerable.rb

```diff
@@ -212,6 +212,12 @@ def slice_when(&block)
     end
   end
 
+  def chunk_while(&block)
+    raise ArgumentError, "wrong number of arguments (0 for 1)" unless block_given?
```

Comment:
This doesn't strike me as a particularly accurate or helpful error message.

Even if it's the same as the one given by MRI, I'd prefer to see it made clear that a *block is required*.  The current error message makes it look like a regular argument is required.